src/styles/main-content.scss

```diff
@@ -0,0 +1,137 @@
+$color_1: #616070;
+$color_2: rgba(6, 11, 53, 1);
+$color_3: white;
+$color_5: rgba(97, 96, 112, 1);
+$font-family_1: "Roboto", sans-serif;
+$light-blue: rgba(0, 172, 220, 1);
+$background-color_2: white;
```

Comment:
now you have duplicated variables - one in variables.scss and one in main-content.scss